server/chat/backend/agent/prompt/background.py

```
@@ -1,71 +1,45 @@
+    except Exception as e:
+        logger.warning(f"Failed to load segment '{segment_name}': {e}")
+        return
+
+    if not content:
         return
 
+    if context:
+        try:
+            content = resolve_template(content, context)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)